src/Logic.cpp

```diff
@@ -141,6 +141,33 @@ void Logic::loadEntities(std::string filename) {
     //entities_["Character"].setVel(sf::Vector2f(1,1));
     addEntity("Character", new Character());
     getCharacter().setVel(sf::Vector2f(1,1));
+    std::ifstream file(filename);
+    std::string str;
+    int counter=1;
+
+    while (std::getline(file, str))
+    {
+        // for each line in the file:
+        // n start_pos.x start_pos.y dest_pos.x dest_pos.y
+        // n is the class of enemy, determine which class of enemy to be created
+        //start_pos and dest_pos are starting and destination, must not be on the wall
+        std::istringstream iss(str);
+        int level,start_x, start_y,dest_x,dest_y;
+        iss >> level >> start_x>> start_y >> dest_x >> dest_y;
+
+        addEntity("Enemy"+std::to_string(counter),new Enemy1());
```

Comment:
Maybe rename the enemy classes to BasicEnemy / AdvancedEnemy, so the numbers don't get confused with these IDs? Also, if the only difference between the enemy types' behavior is numerical, like sight range / speed, then instead of two classes we could have a single class represent both of them, and construct them using a boolean to choose those values and sprites.

Reply:
I think it makes sense, so far I couldn't envision too much differences between Enemy1 and Enemy2 therefore I am constructing a group of enemies only using Enemy1. I think eventually we may just remove enemy2 altogether and rename Enemy1 to just Enemy or something

Reply:
I think a single Enemy class is a good idea